CHANGES.md

```diff
@@ -6,7 +6,7 @@ Version 0.2.0
 
 To be released.
 
- -  The type of `Peer.Urls` property is changed from `Uri` to `IPEndPoint`.
+ - `Peer.Urls` is changed to `Peer.EndPoint` represented as single `IPEndPoint`.
```

Comment:
There were a renaming and a retyping, so we'd better to sum up the explanation at higher view, with a link to the issue #120 and #126.

>  -  Since we decided to depend on TURN ([RFC 5766]) and STUN ([RFC 5389]) to work around NAT so that `Peer`'s endpoints don't have to be multiple, `Peer.Urls` was renamed to `Peer.EndPoint` and its type also was changed from `IImmutableList<Uri>` to `IPEndPoint`.  [#120, #126]

[RFC 5766]: https://tools.ietf.org/html/rfc5766
[RFC 5389]: https://tools.ietf.org/html/rfc5389